docs: initial accessibility section in each component page (#175)
Refinitiv/refinitiv-ui: documents/src/pages/elements/select.md

| ------------------ | ------------------------------------ |
| --list-max-height  | Maximum height of the drop-down list |
| --list-max-width   | Maximum width of the drop-down list  |

## Accessibility
::a11y-intro::

> `ef-select` is assigned `role="button"` and also attribute `aria-expanded`. Select options are assigned `role="option"` and can include attributes such as `aria-label` and `aria-selected`. The element’s state programmatically updates to match its visual state.  

Reviewer: ```suggestion
`ef-select` is assigned `role="button"` and also attribute `aria-expanded`. Select options are assigned `role="option"` and `aria-selected` which depends on item's selection state.
```

* `ef-select` manages the role and aria attributes automatically if you create `ef-select` using `data` property

> * If you create select declaratively by using `ef-item`, assign `role="option"` to each `ef-item`

Reviewer: ```suggestion
* If you create select declaratively by using `ef-item`, assign `role="option"` to selectable `ef-item`.
```

| ------------------ | ------------------------------------ |
| --list-max-height  | Maximum height of the drop-down list |
| --list-max-width   | Maximum width of the drop-down list  |

## Accessibility
::a11y-intro::

> `ef-select` is assigned `role="listbox"` and can include properties such as `aria-expanded` and `aria-activedescendant`. Select options are assigned `role="option"` and can include properties such as `aria-label` and `aria-selected`. The element’s state programmatically updates to match its visual state.  

Reviewer 2: ```suggestion
`ef-select` is assigned `role="button"` and also attribute `aria-expanded`. Select options are assigned `role="option"` and can include attributes such as `aria-label` and `aria-selected`. The element’s state programmatically updates to match its visual state.  
```

Reviewer: `ef-select` is actually assigned `role="button"` and does not use `aria-activedescendant` instead it focuses on item directly.
